api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatient.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * Defines a MpiPatient in the system. A mpi patient is simply an extension of a patient and all that that
+ * implies.
+ */
+public class MpiPatient extends Patient {
```

Comment:
@djazayeri "or better, object representing actual MPI response" we have such object. It calls OpenEmpiPatientQuery.
This inheritance from Patient is important, it provides us to use MPI patients as common patients.